src/sagas/timerSaga.js

```diff
@@ -0,0 +1,19 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+
+import { getTimer } from 'apis/timerApi';
+import { timerAction, timerRequestFailed } from 'actions/timerActions';
+
+import { TIMER } from 'constants/actionConstants';
+
+export function* timerSaga() {
```

Comment:
Codacy has a fix for the issue: [Missing space before *.](https://app.codacy.com/gh/joshsoftware/OnCoT-Frontend/pullRequest?prid=7073195)
```suggestion
export function * timerSaga() {
```